portfolio/about.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+	<meta charset="utf-8">
+	<title>Pushpa's Portfolio Page</title>
+	<link rel="stylesheet" href="Styles/about.css">
+
+</head>
+<body>
+
+	<main class="container">
+
+		<div class="navigator">
+			<nav>
```

Comment:
You're using tabs for indentation rather than spaces. It's important that we all be using the same system so that we can collaborate and share code easily. Please go into your editor's settings and change it to use spaces instead of tabs.